extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -1166,6 +1214,7 @@ public void verifyResults()
         verifyCounters(reportPayload.getCounters());
 
         MSQSpec foundSpec = indexingServiceClient.getMSQControllerTask(controllerId).getQuerySpec();
+        verifyLookupLoadingInfoInTaskContext(indexingServiceClient.getMSQControllerTask(controllerId).getContext());
```

Comment:
The indexing client is already being invoked to get the controller task in the previous statement. Let's just reuse that value.